Source/VCGeneration/Split.cs

```diff
@@ -117,21 +116,21 @@ void ObjectInvariant()
       internal VCGen.ErrorReporter reporter;
 
       public Split(List<Block /*!*/> /*!*/ blocks, Dictionary<TransferCmd, ReturnCmd> /*!*/ gotoCmdOrigins,
-        VCGen /*!*/ par, Implementation /*!*/ impl)
+        VCGen /*!*/ par, Implementation /*!*/ implementation)
       {
         Contract.Requires(cce.NonNullElements(blocks));
         Contract.Requires(gotoCmdOrigins != null);
         Contract.Requires(par != null);
-        Contract.Requires(impl != null);
+        Contract.Requires(implementation != null);
         this.blocks = blocks;
         this.gotoCmdOrigins = gotoCmdOrigins;
         this.parent = par;
-        this.impl = impl;
+        this.Implementation = implementation;
         Interlocked.Increment(ref currentId);
 
-        PrintTopLevelDeclarationsForPruning(impl, "before");        
+        PrintTopLevelDeclarationsForPruning(implementation, "before");        
         TopLevelDeclarations = Prune.GetLiveDeclarations(par.program, blocks).ToList();
-        PrintTopLevelDeclarationsForPruning(impl, "after");
+        PrintTopLevelDeclarationsForPruning(implementation, "after");
```

Comment:
I'm usually not a fan of this kind of renaming, since it pollutes the history of the codebase without much benefit.

Reply:
The field had to capitalise its first letter because it's public now, and then I also made the name longer.